### PR TITLE
feat: Add support from hidden suppressions api

### PR DESCRIPTION
**Pull request by rmoles**
Signed-off-by: Ross <ross.moles@lacework.net>

## Summary

As part of the legacy compliance deprecation effort, the legacy cfg-analyzers will be decommissioned some time in January 2023. As a result customers will be moved across to the new LPP policies and CIS benchmarks. This is likely to cause a lot of noise for customers as they will have suppressions in place for resources which they expect to be in a non compliant state.
At present there is no easy path for customers to migrate their old suppressions to the new exceptions policies.

We do have a scripted approach created by Marc Garcia: https://github.com/lacework-dev/scripts/tree/main/lw-suppressions-migration

However it was decided that we should add this to the CLI as a more "official" approach.

This PR adds support for the hidden suppressions API

## How did you test this change?
Tested locally ✅ 

## Issue
https://lacework.atlassian.net/browse/RAIN-44577

### CODE REVIEW

File: api/v2.go [hunk 1]
```diff
@@ -88,6 +89,11 @@ func NewV2Endpoints(c *Client) *V2Endpoints {
 		&VulnerabilityExceptionsService{c},
 		NewV2VulnerabilitiesService(c),
 		&AlertsService{c},
+		&SuppressionsServiceV2{c,
```
Comment (jon-stewart) [inline api/v2.go:92]: Missing closing bracket here?

**Review by jon-stewart — APPROVED** [review]

File: api/v2.go (hunk 1 above)
Comment (rmoles) [reply api/v2.go:92]: the closing bracket is on line 96

File: api/v2_suppressions_test.go [hunk 2]
```diff
@@ -0,0 +1,839 @@
...
+	expectedRecsWithSup := []string{"AWS_CIS_1_14", "AWS_CIS_2_7", "AWS_CIS_4_4",
+		"LW_AWS_NETWORKING_50",
+		"LW_S3_1"}
+	sort.Strings(expectedRecsWithSup)
+	sort.Strings(recsWithSuppressions)
+	assert.Equal(t, expectedRecsWithSup, recsWithSuppressions)
+}
+
+func generateSuppressions() string {
+	return ListSuppressions()
```
Comment (iok0) [inline api/v2_suppressions_test.go:73]: nit: Could collapse to one method here? I was looking for where ListSuppressions() was used - then I found this wrapper

Comment (rmoles) [reply api/v2_suppressions_test.go:73]: Fair point. I'll get that updated

**Pull request merged**